apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/CommitController.java

```diff
@@ -36,7 +46,18 @@ public CommitController(final CommitService commitService, final PermissionValid
     if (permissionValidator.shouldHideConfigToCurrentUser(appId, env, namespaceName)) {
       return Collections.emptyList();
     }
-
-    return commitService.find(appId, Env.valueOf(env), clusterName, namespaceName, page, size);
+    List<CommitDTO> dtoList = commitService
+        .find(appId, Env.valueOf(env), clusterName, namespaceName, page, size);
+    if (CollectionUtils.isEmpty(dtoList)) {
+      return Collections.emptyList();
+    }
+    Set<String> operatorIdSet = PreferredUsernameUtil.extractOperatorId(dtoList);
```

Comment:
how about we combine these 3 operations into 1  operation? e.g. `PreferredUsernameUtil.enrichPreferredUserName(dtoList);`

Reply:
and how about we move this enrich logic to commitService?